### PR TITLE
Show editor config tip when running `state use show`

### PR DESCRIPTION
**Pull request by Naatan**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1238" title="DX-1238" target="_blank">BugDX-1238</a>  USE: Can't set the project as default
  </td></table>
  <br />

### CODE REVIEW

File: test/integration/use_int_test.go [hunk 1]
```diff
@@ -195,6 +197,8 @@ func (suite *UseIntegrationTestSuite) TestShow() {
 		suite.Require().NoError(err)
 		cp.ExpectLongString(longPath)
 	}
+	cp.ExpectLongString(ts.Dirs.Cache)
+	cp.Expect("/exec")
```
Comment (mitchell-as) [inline test/integration/use_int_test.go:201]: Is this going to work on Windows?

Comment (Naatan) [reply test/integration/use_int_test.go:201]: Will run the test on Windows. IIRC we run them in bash on Windows so in theory it should work.

**Review by mitchell-as — CHANGES_REQUESTED** [review]
This looks good other than the failing integration test on Windows.

**Review by mitchell-as — APPROVED** [review]

**Comment by Naatan** [comment]
Reopening with the correct Jira ID.

**Comment by Naatan** [comment]
https://github.com/ActiveState/cli/pull/2108

**Pull request closed without merging**